src/from_sql.rs

```diff
@@ -2,6 +2,33 @@ use crate::{tds::Numeric, xml::XmlData, ColumnData};
 use uuid::Uuid;
 
 /// A conversion trait from a TDS type by-reference.
+///
+/// An implementation to a Rust type is needed for using it as a return
+/// parameter from [`Row#get`] or [`Row#try_get`] methods. The following Rust
+/// types are already implemented to match the given server types:
+///
+/// |Rust type|Server type|
+/// |--------|--------|
+/// |`i8`|`tinyint`|
+/// |`i16`|`smallint`|
+/// |`i32`|`int`|
+/// |`i64`|`bigint`|
+/// |`f32`|`float(24)`|
+/// |`f64`|`float(53)`|
+/// |`bool`|`bit`|
+/// |`String`/`&str`|`nvarchar`/`varchar`/`nchar`/`char`/`ntext`/`text`|
+/// |`Uuid`|`uniqueidentifier`|
+/// |`Vec<u8>`/`&[u8]`|`binary`/`varbinary`/`image`|
+/// |`Numeric`|`numeric`/`decimal`|
+/// |`Decimal`|`numeric`/`decimal`|
+/// |`XmlData`|`xml`|
+/// |`NaiveDateTime`|`datetime`/`datetime2`/`smalldatetime`|
+/// |`NaiveDate`|`date`|
+/// |`NaiveTime`|`time`|
```

Comment:
Where does `NaiveDateTime` come from? Do we re-export these?

Reply:
This is the question. The [time module](https://docs.rs/tiberius/0.4.4/tiberius/time/index.html) explains that one can in theory use the provided date and time types, but they are quite useless and the preferred way is to use the `chrono` feature flag with `chrono` types if needing to manipulate any of these values.

Reply:
Should we re-export these btw?

Reply:
yeah, I think so -- they're a peer dependency already, so may as well make it easier to use (and ensure people don't run into version mismatches).